include/macros.h

```diff
@@ -26,6 +26,8 @@
 
 #define PLAYER ((Player*)globalCtx->actorCtx.actorList[ACTORCAT_PLAYER].first)
 
+#define CUR_TRANSFORMATION ((gSaveContext.transformation == 4) ? 0 : gSaveContext.transformation)
```

Comment:
```suggestion
#define CUR_TRANSFORMATION ((gSaveContext.transformation == PLAYER_FORM_HUMAN) ? PLAYER_FORM_FIERCE_DEITY : gSaveContext.transformation)
```

Reply:
I don't think that makes sense: isn't the point of this macro to convert from transformation index to button equips index?

Reply:
Just want to note that I've seen 2 different ways where the transformation index gets converted to an array index (both used in EnToto). Either with this macro where human gets wrapped to 0 or just by subtracting 1. Don't know if we would even want to macro the second case, just wanted to bring it up as it could be relevant.